docs/docs/setting-up/networking-instructions/networking.md

```diff
@@ -3,25 +3,29 @@ sidebar_position: 4
 sidebar_label: "Internet Access"
 ---
 
-# Accessing the Internet from Jobs
+# Accessing the Network From Jobs
 
-By default, Bacalhau jobs do not have any access to the internet. This is to keep both compute providers and users safe from malicious activities.
+By default, after launch Bacalhau jobs do not have any access to the external network - Internet or other nodes in the same network. This is to keep both compute providers and users safe from malicious activities.
 
-However, by using data volumes you can read and access your data from within jobs and write back results.
+However, there are 2 ways to access your data - before or during the execution of a job:
+- Using Data Volumes to download the input data and upload the results
+- Using `--network` flag to allow job to access the internet 
+:::info
+For more details about flags and recent updates see the [CLI Guide](../../dev/cli-reference/all-flags.md) or execute `bacalhau help`. Also feel free to contact us on [Slack](https://bacalhauproject.slack.com).
+:::
 
 ## Using Data Volumes
 
-When you submit a Bacalhau job, you'll need to specify the internet locations to download data from and write results to. Both [Docker](../getting-started/docker-workload-onboarding.md) and [WebAssembly](../getting-started/wasm-workload-onboarding.md) jobs support these features.
+If you need to process data located on a certain network resource and/or save the result of the job execution to such resource - you will need to specify the internet locations to download data from and write results to when creating the job. Both [Docker](../../getting-started/docker-workload-onboarding.md) and [WebAssembly](../../getting-started/wasm-workload-onboarding.md) jobs support these features.
```

Comment:
We might want to skip discussing saving the result of the job in this paragraph as it's mentioned in a paragraph below.  Perhaps this paragraph could just talk about downloading data?

Reply:
fixed